client/service-container-chain/src/data_preservers.rs

```diff
@@ -37,6 +37,8 @@ pub async fn task_watch_assignment(spawner: impl Spawner, profile_id: ProfileId)
     use dc_orchestrator_chain_interface::DataPreserverAssignment as Assignment;
 
     if let OrchestratorChainResult::Err(e) = try_fut(async move {
+        log::info!("Starting data preserver assignment watcher");
```

Comment:
does this log in every block import? if so we maybe should consider moving it to debug?

Reply:
It logs only at the start of data preserver mode. I can move it to debug tho.